### PR TITLE
jicofo & jvb: Fix OCTO + SCTP behaviour

### PR DESCRIPTION
**Pull request by spijet**
Hello everyone!

As a follow-up to #1787, here's the PR that:
1. Removes the bogus `cont-init` check;
2. Enables SCTP by default in Jicofo config template.

I'll leave the PR as a draft for now, until @saghul confirms that all that needed to be done is done. :)

### CODE REVIEW

File: jicofo/rootfs/defaults/jicofo.conf [hunk 1]
```diff
@@ -4,10 +4,10 @@
 {{ $AUTH_TYPE := .Env.AUTH_TYPE | default "internal" -}}
 {{ $JICOFO_AUTH_TYPE := .Env.JICOFO_AUTH_TYPE | default $AUTH_TYPE -}}
 {{ $JICOFO_AUTH_LIFETIME := .Env.JICOFO_AUTH_LIFETIME | default "24 hours" -}}
-{{ $ENABLE_SCTP := .Env.ENABLE_SCTP | default "0" | toBool -}}
+{{ $ENABLE_SCTP := .Env.ENABLE_SCTP | default "1" | toBool -}}
 {{ $ENABLE_RECORDING := .Env.ENABLE_RECORDING | default "0" | toBool -}}
 {{ $ENABLE_OCTO := .Env.ENABLE_OCTO | default "0" | toBool -}}
-{{ $ENABLE_OCTO_SCTP := .Env.ENABLE_OCTO_SCTP | default (.Env.ENABLE_SCTP | default "0") | toBool -}}
+{{ $ENABLE_OCTO_SCTP := .Env.ENABLE_OCTO_SCTP | default (.Env.ENABLE_SCTP | default "1") | toBool -}}
```
Comment (saghul) [inline jicofo/rootfs/defaults/jicofo.conf:10]: I think you can simplify this by defaulting to $ENABLE_SCTP

**Comment by saghul** [comment]
@damencho can you PTAL too?

**Comment by damencho** [comment]
> @damencho can you PTAL too?

I asked @bgrozev and he said it is fine :)

File: jicofo/rootfs/defaults/jicofo.conf (hunk 1 above)
Comment (spijet) [reply jicofo/rootfs/defaults/jicofo.conf]: Is it better now? :)

**Comment by spijet** [comment]
I updated the config template according to @saghul's comment and switched this PR from draft to open. Please let me know if there's anything else that needs to be added. :)

File: jicofo/rootfs/defaults/jicofo.conf [hunk 2]
```diff
@@ -4,10 +4,10 @@
 {{ $AUTH_TYPE := .Env.AUTH_TYPE | default "internal" -}}
 {{ $JICOFO_AUTH_TYPE := .Env.JICOFO_AUTH_TYPE | default $AUTH_TYPE -}}
 {{ $JICOFO_AUTH_LIFETIME := .Env.JICOFO_AUTH_LIFETIME | default "24 hours" -}}
-{{ $ENABLE_SCTP := .Env.ENABLE_SCTP | default "0" | toBool -}}
+{{ $ENABLE_SCTP := .Env.ENABLE_SCTP | default "1" | toBool -}}
```
Comment (saghul) [inline jicofo/rootfs/defaults/jicofo.conf:7]: Upon further inspection I think we should keep this defaulting to disabled, for the time being. With the rest of the changes inplace you'll still be able to enable OCTO over SCTP though.

Comment (spijet) [reply jicofo/rootfs/defaults/jicofo.conf]: Done in b2ecc27. :)

**Review by saghul — APPROVED** [review]

**Comment by spijet** [comment]
Yay! ❤️

**Pull request merged**